packages/bsky/src/services/indexing/index.ts

```diff
@@ -93,6 +103,57 @@ export class IndexingService {
     }
   }
 
+  async indexRepo(did: string, commit: string) {
+    const now = new Date().toISOString()
+    const { pds, signingKey } = await this.didResolver.resolveAtpData(did)
+    const { sync } = new AtpAgent({ service: pds }).api.com.atproto
+
+    const { data: car } = await sync.getCheckout({ did, commit })
+    const { root, blocks } = await readCarWithRoot(new Uint8Array(car))
```

Comment:
after lex refactor this'll actually be returning the right type & you won't have to wrap in `uint8array`!